app/templates/confirmation.html

```diff
@@ -7,8 +7,11 @@
 <a class="u-dib" href="{{previous_location}}">Previous</a>
 
 <div class="header">
-  <h1 class="header__title saturn">Confirmation</h1>
-  <h2 class="header__subtitle">Thank you for your answers, do you wish to submit</h2>
+  <h1 class="header__title saturn">You’ve completed the 2017 Census Test</h1>
+  <h2 class="header__subtitle">Submission</h2>
+    <ul>
```

Comment:
Is this really meant to be here? Seems like the confirmation page is being updated in this PR: https://github.com/ONSdigital/eq-survey-runner/pull/688/files, and that's different to this?